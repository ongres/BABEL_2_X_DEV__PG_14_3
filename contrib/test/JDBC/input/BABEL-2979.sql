alter user john with login = smith
GO
