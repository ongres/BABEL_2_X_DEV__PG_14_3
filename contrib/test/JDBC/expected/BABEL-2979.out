alter user john with login = smith
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: 'ALTER USER WITH LOGIN' is not currently supported in Babelfish)~~

